lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
@pmuens I created an example CF stack and added [a question](https://forums.aws.amazon.com/thread.jspa?messageID=787503&#787503) about removal to the AWS Developer Forum.

It seems that it is not normal behavior that Usage Plan prevents stack deletion and they will look into it. I don't know how soon they can provide a solution, though.